### DISCUSSION TITLE
how make graphql gateway ?

### DISCUSSION THREAD

**Discussion by rullyalves** [ExpediaGroup/graphql-kotlin#1473] [General]
hi, thanks for the great work

is it possible to use create a graphql gateway just using the library with kotlin or are there only implementations of the graphql gateway for javascript/nodejs as in the federation example ?

**Comment by samuelAndalon** [comment]
hello, the schema generator will create a schema compatible [with Apollo Federation spec](https://www.apollographql.com/docs/federation/federation-spec/), this library does not include (yet) an implementation of the gateway.
